stable/sysdig/templates/secrets.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
```

Comment:
Would you like to add a consistent label set to all resources created by this chart?